Adding custom build system

aaronbell opened the pull request:
Added a couple of build files necessary to get the font produced for GF's purposes. Pretty much the same system as in Chiron Hei HK.

aaronbell committed:
Adding necessary bits for building the GF version of the fonts.

aaronbell committed:
Create ChironGoRoundTC[wght].ttf

GF built version

aaronbell committed:
Update gf_notdef.py

aaronbell committed:
Updating build script

Now corrects two additional bugs and adds the vhea / vmtx tables.

aaronbell commented:
I've found some additional items I want to review and work through tomorrow.

tamcy commented:
Sorry for the late reply—I’ve been busy preparing a new version of the font over the past few days, using the just-released Source Han Sans / Noto Sans CJK as the base, along with some additional bug fixes. If possible, I’d suggest waiting for the next release (which should be ready soon) before proceeding. What do you think?

aaronbell commented:
@tamcy Yup, no problem. As part of this, I also discovered that the vhea / vmtx tables are not being built by the system for Chiron Hei (in either your version or mine, it seems), so will need to go fix that one up too. I'll do the rebuild once you've updated to the new release.

tamcy commented:
@aaronbell Thank you! I have committed the latest source files.

aaronbell committed:
Fixed a couple things for FB

- Added glyph height, and vhea info to ensure generation of the vmtx / vhea tables
- Also added NID 9, which was not being converted.

aaronbell commented:
I think this is looking ready for consideration. Now getting a vmtx / vhea table!

tamcy commented:
Thanks @aaronbell for the help!
